docker-compose.yml

```diff
@@ -1,14 +1,11 @@
-version: '3.5'
+version: "3"
+
 services:
   imgpush:
+    container_name: imgpush   
+    image: hauxir/imgpush:latest
+    restart: unless-stopped
     ports:
-        - "5000:5000"
-    build: .
+      - "5000:5000/tcp"
     volumes:
-      - ./app:/app
-      - ./images:/images
-      - ./cache:/cache
-    healthcheck:
-      start_period: 0s
-      test: ['CMD-SHELL', 'curl localhost:5000/liveness -s -f -o /dev/null || exit 1']
-      interval: 30s
+      - './config:/images'
```

Comment:
/config? what's wrong with the current volumes?

Reply:
There isn't anything wrong, but your docker-compose.yml file made the user build their own docker image. Also, a user made a issue about the healthcheck not working, hence why I removed it.

Reply:
So, will you pull this?

Reply:
> /config? what's wrong with the current volumes?

Current volumes are exposing /app folder to the host unnecessarily when it's already copying to the container on the Dockerfile. This also produces the "entrypoint not found" when you run this image without cloning the repo.

Cache doesn't make much sense exposing to host if you're not troubleshooting problems.

Most people here are not using Kubernetes or any type of cluster system, they're running this as a standalone "Minimalist Self-hosted Image Service" for their projects at home I assume.